# Joint limits optimized for 5-DOF planning
# More aggressive limits help the planner find solutions faster

default_velocity_scaling_factor: 0.3  # Increased from 0.1 for smoother motion
default_acceleration_scaling_factor: 0.3

joint_limits:
  Joint_1:
    has_velocity_limits: true
    max_velocity: 2.0  # rad/s - reasonable for safe operation
    has_acceleration_limits: true
    max_acceleration: 1.0
  Joint_2:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  Joint_3:
    has_velocity_limits: true
    max_velocity: 2.0
    has_acceleration_limits: true
    max_acceleration: 1.0
  Joint_4:
    has_velocity_limits: true
    max_velocity: 2.5  # Wrist joints can usually move faster
    has_acceleration_limits: true
    max_acceleration: 1.5
  Joint_5:
    has_velocity_limits: true
    max_velocity: 2.5
    has_acceleration_limits: true
    max_acceleration: 1.5
  Joint_EE:
    has_velocity_limits: true
    max_velocity: 1.0
    has_acceleration_limits: true
    max_acceleration: 0.5
